src/styles/blocks/card.scss

```
@@ -0,0 +1,109 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  font-family: Roboto, sans-serif;
+}
```

don't use positioning on **body** tag. Use a separate class - `.body` for it
```suggestion
body {
  margin: 0;
  padding: 0;
  font-family: Roboto, sans-serif;
}
```